layers/+completion/spacemacs-helm/packages.el

```diff
@@ -956,3 +957,22 @@ Search for a search tool in the order provided by `dotspacemacs-search-tools'."
     :init
     (spacemacs/set-leader-keys
       "Th" 'helm-themes)))
+
+(defun spacemacs-helm/init-swiper-helm ()
+  (use-package swiper-helm
+    :defer t
+    :init
+    (progn
+
+      (defun spacemacs/swiper-helm-region-or-symbol ()
+        (interactive)
+        (swiper-helm
+         (if (region-active-p)
```

Comment:
I copy-pasted the code above because I was not sure where to extract a private function (like `spacemacs--region-or-symbol-at-point`). Thoughts?